docs/architecture/grpc-for-wcf-developers/protobuf-messages.md

```diff
@@ -1,11 +1,80 @@
 ---
 title: Protobuf messages - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: How Protobuf messages are defined in the IDL and generated in C#
 author: markrendle
-ms.date: 09/02/2019
+ms.date: 09/09/2019
 ---
 
 # Protobuf messages
 
+In WCF, a `Stock` class for a stock market trading application might be defined like this.
+
+```csharp
+[DataContract]
+public class Stock
+{
+    [DataMember]
+    public int Id { get; set;}
+    [DataMember]
+    public string Symbol { get; set;}
+    [DataMember]
+    public decimal Price { get; set;}
+    [DataMember]
+    public string DisplayName { get; set;}
+}
+```
+
+To implement the equivalent class in Protobuf, it must be declared in the `.proto` file. The `protoc` compiler will then generate the .NET class as part of the build process.
+
+```protobuf
+syntax "proto3";
+
+message Stock {
+
+    int32 id = 1;
+    string symbol = 2;
+    double price = 3;
```

Comment:
Floating point value for money isn't a good idea 😬 

Edit: I see there is a note below calling out the problem, but I would consider choosing a different type to demo. Should avoid demoing bad practices.

Reply:
Yeah, I know. I'll change it.